psamm_import/datasource/cobrajson.py

```diff
@@ -106,16 +106,24 @@ def _read_compounds(self, doc):
             yield CompoundEntry(id=id, name=name, charge=charge,
                                 formula=formula)
 
-    def _parse_reaction_equation(self, doc):
+    def _parse_reaction_equation(self, r):
         compounds = ((Compound(metabolite), value)
-                     for metabolite, value in iteritems(doc))
-        return Reaction(Direction.Both, compounds)
+                     for metabolite, value in iteritems(r['metabolites']))
+        if (r.get('lower_bound') == 0 and
+           r.get('upper_bound') != 0):
```

Comment:
There's some slight misalignment of the indent here. Other than that, this looks great!